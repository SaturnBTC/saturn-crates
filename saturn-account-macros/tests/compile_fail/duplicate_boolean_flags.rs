use saturn_account_macros::Accounts;

#[derive(Accounts)]
struct DupFlags<'info> {
    #[account(signer, signer)]
    user: arch_program::account::AccountInfo<'info>,
}

fn main() {} 