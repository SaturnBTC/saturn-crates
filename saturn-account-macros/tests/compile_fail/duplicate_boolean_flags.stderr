error: duplicate `signer` flag
 --> tests/compile_fail/duplicate_boolean_flags.rs:5:23
  |
5 |     #[account(signer, signer)]
  |                       ^^^^^^
